scaffolder-templates/clean-architecture-app/template.yaml

```diff
@@ -0,0 +1,133 @@
+apiVersion: scaffolder.backstage.io/v1beta3
+# https://backstage.io/docs/features/software-catalog/descriptor-format#kind-template
+kind: Template
+metadata:
+  name: scaffold-bancolombia-java
+  title: Clean Architecture Java Template
+  description: Template for the scaffolder that creates a clean architecture project java
+  tags:
+    - recommended
+    - java
+spec:
+  owner: user:guest
+  type: service
+
+  # These parameters are used to generate the input form in the frontend, and are
+  # used to gather input data for the execution of the template.
+  parameters:
+    - title: Fill in some steps
+      required:
+        - name
+        - package
+        - projectType
+        - coverage
+        - owner
+      properties:
+        name:
+          title: Name Application
+          type: string
+          description: Unique name of the component
+          ui:autofocus: true
+          ui:options:
+            rows: 5
+        package:
+          title: co.com.bancolombia
+          type: string
+          description: You can specify the main or default package of your project
+          ui:autofocus: true
+          ui:options:
+            rows: 5
+        projectType:
+          title: Type imperative - reactive
+          type: string
+          description: With this parameter the task will generate a POO project
+          ui:autofocus: true
+          ui:options:
+            rows: 5
+        coverage:
+          title: Coverge jacoco - cobertura
+          type: string
+          description: This parameter is going to specify the coverage tool for the project
+          ui:autofocus: true
+          ui:options:
+            rows: 5
+        lombok:
+          title: With lombok
+          type: boolean
+          description: Select to use lombok
+          default: false
+          ui:autofocus: true
+          ui:options:
+            rows: 5
+        metrics:
+          title: With metrics
+          type: boolean
+          description: Select to enable metrics
+          default: false
+          ui:autofocus: true
+          ui:options:
+            rows: 5
+        owner:
+          title: Owner
+          type: string
+          description: Owner of the component
+          ui:field: OwnerPicker
+          ui:options:
+            allowedKinds:
+              - Group
+    - title: Choose a location
+      required:
+        - repoUrl
+      properties:
+        repoUrl:
+          title: Repository Location
+          type: string
+          ui:field: RepoUrlPicker
+          ui:options:
+            allowedHosts:
+              - github.com
+              - dev.azure.com
+
+  # These steps are executed in the scaffolder backend, using data that we gathered
+  # via the parameters above.
+  steps:
+    # Each step executes an action, in this case one templates files into the working directory.
+    - id: fetch-base
+      name: Fetch Base
+      action: fetch:template
```

Comment:
Since you have a `.github` folder, you might wanna copy it as it is. An example https://github.com/backstage/software-templates/blob/41ee7e340415ba4d846d378a8674c8981d5cfc9d/scaffolder-templates/react-ssr-template/template.yaml#L53-L54

Reply:
Aah, my bad. I see you have used the template variables as well in the gradle step, so you can ignore my comment. :)

Reply:
Thank you for your comments!! I hope this template was clear and easy to use, can you add me the "hacktoberfest-accepted" label if the PR is accepted?